xcode/Frameworks/JoyStickView/JoyStickView.xcodeproj/xcuserdata/mpe18.xcuserdatad/xcschemes/xcschememanagement.plist

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@mist64
Having support for a build system such as cmake or meson would obviate the need for inclusion of these project files.